Dockerfile

```diff
@@ -14,6 +17,8 @@ RUN yum -y install gcc-c++ && \
     yum clean all && \
     rm --force ns.rpm
 
+RUN echo complete -C '/usr/bin/aws_completer' aws >> /root/.bashrc
```

Comment:
yes indeed, this is a personal preference. If I use the docker image for deployment I will also run `docker run -it amazonlinux:nodejs bash` and then `aws cloudformation list-stacks` and so completion would we nice.
Should I remove it for the PR to pass? Sorry I am very new to this Github PR stuff.